subworkflows/nf-core/variantcalling/manta/tumoronly/main.nf

```diff
@@ -48,7 +55,14 @@ workflow RUN_MANTA_TUMORONLY {
     MERGE_MANTA_SV(
         manta_candidate_sv_vcf.intervals.map{ meta, vcf ->
 
-                [groupKey([patient:meta.patient, sample:meta.sample, status:meta.status, sex:meta.sex, id:meta.sample, num_intervals:meta.num_intervals],
+                [groupKey([
+                            id:             meta.sample,
+                            num_intervals:  meta.num_intervals,
+                            patient:        meta.patient,
+                            sample:         meta.sample,
+                            sex:            meta.sex,
+                            status:         meta.status
```

Comment:
Also wondering if the `variantcaller:"manta"` has to be added to al manta-related processes, but not sure 🤔

Reply:
It is added in the final formatting for the output. Here in these line the meta map is just adapted generate the groupKey for groupTuple on variable input